script/bootstrap

```diff
@@ -60,6 +60,20 @@ setup_gitconfig () {
   fi
 }
 
+setup_git_symlinks () {
+  if ! [ $(readlink ~/.gitconfig) = */.dotfiles/developer/common/git/gitconfig.symlink ] &&
+   [ $(readlink ~/.gitconfig.local) = */.dotfiles/developer/common/git/gitconfig.local.symlink ] &&
+   [ $(readlink ~/.gitignore) = */.dotfiles/developer/common/git/gitignore.symlink ]
+  then
+    echo "Symlinks correct"
```

Comment:
No need for this. We can silently move on if all is 👍.

Reply:
check!